### PR TITLE
GH1591: Add ChocolateyDownload alias

### LINKED ISSUE #1591: Add support for choco download
Add support for `choco download` which would allow to use a Cake script for [internalizing Chocolatey packages](https://chocolatey.org/docs/features-automatically-recompile-packages) from the community feed and publish them to an internal feed.

### PR DESCRIPTION
**Pull request by pascalberger**
Add alias for `choco download`. Mentionend in the XML comments functionality which requires Chocolatey busines or licensed editions.

Fixes #1591.

### CODE REVIEW

File: src/Cake.Common/Tools/Chocolatey/Download/ChocolateyDownloader.cs [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        {
+            _environment = environment;
+        }
+
+        /// <summary>
+        /// Installs Chocolatey packages using the specified package configuration file and settings.
+        /// </summary>
+        /// <param name="packageConfigPath">Path to package configuration to use for install.</param>
+        /// <param name="settings">The settings.</param>
+        public void InstallFromConfig(FilePath packageConfigPath, ChocolateyDownloadSettings settings)
```
Comment (gep13) [inline src/Cake.Common/Tools/Chocolatey/Download/ChocolateyDownloader.cs]: @pascalberger what is this for?

Comment (pascalberger) [inline src/Cake.Common/Tools/Chocolatey/Download/ChocolateyDownloader.cs]: @gep13 Oops, shouldn't be there. Removed it.

**Comment by pascalberger** [comment]
@gep13 This should be ready for another review

**Comment by gep13** [comment]
@pascalberger this looks good to me.

I "think" the plan just now is to complete the 0.20.0 release, which is a major chunk of work, and then start pulling in other new functionality.

**Comment by pascalberger** [comment]
@gep13 Any chance to get this in now that 0.20.0 is released? Anything I should do (rebase, etc)?

**Comment by devlead** [comment]
@pascalberger LGTM, I'll do a final review post CI finishes and then merge if all's ok.

**Comment by devlead** [comment]
@pascalberger  your changes have been merged, thanks for your contribution 👍

**Pull request merged**